docs/core/diagnostics/sos-debugging-extension.md

```diff
@@ -0,0 +1,369 @@
+
+```
+!DumpVC 0090320c 00a79d9c
+```
+
+ The following command displays the process memory used by the garbage collector.
+
+```
+!eeheap -gc
+```
+
+ The following command displays all objects scheduled for finalization.
+
+```
+!finalizequeue
+```
+
+ The following command determines the application domain of an object at the address `00a79d98`.
+
+```
+!findappdomain 00a79d98
+```
+
+ The following command displays all garbage collector handles in the current process.
+
+```
+!gcinfo 5b68dbb8
+```
+
+ The following command displays the `MethodTable` and `EEClass` structures for the `Main` method in the class `MainClass` in the module `unittest.exe`.
+
+```
+!name2ee unittest.exe MainClass.Main
+```
+
+ The following command displays information about the metadata token at the address `02000003` in the module `unittest.exe`.
+
+```
+!token2ee unittest.exe 02000003
+```
+
+### LLDB Example Usage
+
+The following command displays the contents of an array at the address `00ad28d0`.  The display starts from the second element and continues for five elements.
+
+```
+sos DumpArray -start 2 -length 5 -detail 00ad28d0
+```
+
+ The following command displays the contents of an assembly at the address `1ca248`.
+
+```
+sos DumpAssembly 1ca248
+```
+
+ The following command displays information about the garbage collector heap.
+
+```
+dumpheap
+```
+
+ The following command writes the contents of the in-memory stress log to a (default) file called StressLog.txt in the current directory.
+
+```
+dumplog
+```
+
+ The following command displays the `MethodDesc` structure at the address `902f40`.
+
+```
+dumpmd 902f40
+```
+
+ The following command displays information about a module at the address `1caa50`.
+
+```
+dumpmodule 1caa50
+```
+
+ The following command displays information about an object at the address `a79d40`.
+
+```
+dumpobj a79d40
+```
+
+ The following command displays the fields of a value class at the address `00a79d9c` using the method table at the address `0090320c`.
+
+```
+sos DumpVC 0090320c 00a79d9c
+```
+
+ The following command displays the process memory used by the garbage collector.
+
+```
+eeheap -gc
+```
+
+ The following command determines the application domain of an object at the address `00a79d98`.
+
+```
+sos FindAppDomain 00a79d98
+```
+
+ The following command displays all garbage collector handles in the current process.
+
+```
+sos GCInfo 5b68dbb8
+```
+
+ The following command displays the `MethodTable` and `EEClass` structures for the `Main` method in the class `MainClass` in the module `unittest.exe`.
+
+```
+name2ee unittest.exe MainClass.Main
+```
+
+ The following command displays information about the metadata token at the address `02000003` in the module `unittest.exe`.
+
+```
+sos Token2EE unittest.exe 02000003
+```
+
+ The following displays the managed threads and the threadstate for one:
```

Comment:
This one wouldn't fit the table so just keep it separate.

Reply:
I removed the block section and changed it to a table.  Please let me know if I should revert the change back to the original format.